apps/web/google_docs.talon

```diff
@@ -0,0 +1,326 @@
+browser.host: docs.google.com
+-
+
+tag(): user.line_commands
+
+file menu | menu file: key(ctrl-alt-f)
+open: key(cmd-o)
+[open] (revision | version) history [open]: key(cmd-alt-shift-h)
+print: key(cmd-p)
+save: key(cmd-s)
```

Comment:
We already have `file save`:

https://github.com/knausj85/knausj_talon/blob/4d39608dc9979017d7a0218b195feb54ff136ffa/core/edit/standard.talon#L15